.travis.yml

```diff
@@ -4,6 +4,7 @@ php:
   - 7.1
   - 7.2
   - 7.3
+  - 8.0
```

Comment:
Oh wow I didn't realize we still use Travis for this! :) Of course it's an open source project so this is all fine.